packages/ffe-system-message-react/src/index.d.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-export interface SystemMessageProps {
```

Comment:
usen takk for ditt bidrag.
Da kan man fjerne className iaf som er en del av React.ComponentProps<'div'>

Ser også ut som vi ikke har en ref. Så det blir vell mest riktigt her att bruke ComponentPropsWithoutRef. tenker hele kan vare slik. enig?

export interface SystemMessageProps extends React.ComponentPropsWithoutRef<'div'>  {
    animationLengthMs?: number;
    icon?: React.ReactNode;
    locale?: 'en' | 'nb' | 'nn';
    onClose?: React.MouseEventHandler<HTMLButtonElement>;
    onColoredBg?: boolean;
    children: React.ReactNode;
}